src/codegen/riscv32/constants-riscv32.h

```diff
@@ -505,19 +482,6 @@ enum Opcode : uint32_t {
   RO_AMOMINU_W = AMO | (0b010 << kFunct3Shift) | (0b11000 << kFunct5Shift),
   RO_AMOMAXU_W = AMO | (0b010 << kFunct3Shift) | (0b11100 << kFunct5Shift),
 
-  // RV64A Standard Extension (in addition to RV32A)
-  RO_LR_D = AMO | (0b011 << kFunct3Shift) | (0b00010 << kFunct5Shift),
```

Comment:
RV64A instr don't be deleted form assembler-riscv32.cc, so don't delete these.Let us open a new a pr to delete it.

Reply:
How about deleting RV64A in this PR? I build it without errors, and d8 works as normal.